lib/discovery/index.js

```diff
@@ -1203,21 +1205,21 @@ export class Topic extends TypedEmitter {
    * @param {TopicServiceStatus} [options.dhtStatus]
    * @param {TopicServiceStatus} [options.mdnsStatus]
    * @param {MdnsDiscovery} [options.mdns]
-   * @param {Hyperswarm} [options.dht]
+   * @param {import('hyperswarm').PeerDiscoverySession} [options.dht]
```

Comment:
based on usage, guessed that it was the discovery session we want to pass here, not the hyperswarm instance itself

Reply:
ah yeah, that looks right